app/bundles/EmailBundle/EventListener/ConfigSubscriber.php

```diff
@@ -77,6 +93,41 @@ public function onConfigBeforeSave(ConfigEvent $event)
             }
         }
 
+        $data['mailer_dsn'] = MailerDsnConvertor::convertArrayToDsnString($data, $this->transportType->getTransportDsnConvertors());
+
+        // remove options that are now part of the DSN string
+        $mailerKeys = array_filter($data, function ($key) {
+            return 0 === strpos($key, 'mailer_option');
+        }, ARRAY_FILTER_USE_KEY);
```

Comment:
Just a nitpick, but we can use arrow functions now
```suggestion
        $mailerKeys = array_filter($data, fn ($key) => 0 === strpos($key, 'mailer_option'), ARRAY_FILTER_USE_KEY);
```

Reply:
Implemented